src/components/LeafletMap.js

```diff
@@ -257,4 +261,12 @@ export class LeafletMap {
       }
     });
   };
+
+  toggleLoadingIndicator = (isLoading) => {
```

Comment:
@timhitchins this could just be an `if/else` rather than short circuit with a `return`, IMO the logic is more clear that way, but just a suggestion.

Reply:
bad habit.  good point.